CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.12.2
+
+### Runtime
+
+* Check for non-nil container after match [#26049](https://github.com/docker/docker/pull/26049)
+* Add retry logic during aufs unmount [#26536](https://github.com/docker/docker/pull/26536)
+* Prevent libdevmapper from deleting /dev/mapper/docker-xxx symbolic links in RemoveDeviceDeferred [#24740](https://github.com/docker/docker/pull/24740)
+* Fixing Regression in exit codes [#26777](https://github.com/docker/docker/pull/26777)
+* Prevent stdout / stderr race condition in limitedBuffer [#26507](https://github.com/docker/docker/pull/26507)
+* Fix exec's children handling [#26874](https://github.com/docker/docker/pull/26874)
+* Fix exec form of HEALTHCHECK CMD [#26208](https://github.com/docker/docker/pull/26208)
+
+### Networking
+* Vendor libnetwork [#26879](https://github.com/docker/docker/pull/26879) [#26953](https://github.com/docker/docker/pull/26953)
+ * Avoid returning early on agent join failures [docker/libnetwork#1473](https://github.com/docker/libnetwork/pull/1473)
+
+### Swarm Mode
+
+* Do not autodetect advertise address on join [#26211](https://github.com/docker/docker/pull/26211)
+* Vendor SwarmKit [#26765](https://github.com/docker/docker/pull/26765)
+ * Bounce session after failed status update [docker/swarmkit#1539](https://github.com/docker/swarmkit/pull/1539)
+ * Fix possible deadlocks [docker/swarmkit#1537](https://github.com/docker/swarmkit/pull/1537)
+ * Update boltdb dependency [docker/swarmkit#1510](https://github.com/docker/swarmkit/pull/1510)
```

Comment:
This isn't technically related to revendoring swarmkit. There was an engine PR that did it as well.